app/views/shared/_dates_and_assignment_header.html.erb

```diff
@@ -36,7 +36,7 @@
       </p>
     <% end %>
 
-    <% if @planning_application.publishable? %>
+    <% if @planning_application.publishable? && !@planning_application.pre_application? %>
```

Comment:
I thought for a moment that it would be good to just modify `publishable?` so that it returns false for preapps, but it looks like that's just delegated to the application type feature. That makes me wonder if the application type is configured wrongly, then?